src/Provider/Github.php

```diff
@@ -62,6 +62,29 @@ public function getResourceOwnerDetailsUrl(AccessToken $token)
         return $this->domain . '/api/v3/user';
     }
 
+    protected function fetchResourceOwnerDetails(AccessToken $token)
+    {
+        $response = parent::fetchResourceOwnerDetails($token);
+
+        if (($response['email'] ?? null) == null) {
```

Comment:
```suggestion
        if (! ($response['email'] ?? null)) {
```

Reply:
Actually this can't be used, because we support PHP 7.3+ with this package.
```suggestion
        if (empty($response['email'])) {
```